pop-api/src/v0/assets/fungibles.rs

```diff
@@ -0,0 +1,408 @@
+	assets::cancel_approval(id, spender)?;
+	assets::approve_transfer(id, spender, value)
+}
+
+/// Increases the allowance of a spender.
+///
+/// # Arguments
+/// * `id` - The ID of the asset.
+/// * `spender` - The account that is allowed to spend the tokens.
+/// * `value` - The number of tokens to increase the allowance by.
+///
+/// # Returns
+/// Returns `Ok(())` if successful, or an error if the operation fails.
+#[inline]
+pub fn increase_allowance(id: AssetId, spender: AccountId, value: Balance) -> Result<()> {
+	assets::approve_transfer(id, spender, value)
+}
+
+/// Decreases the allowance of a spender.
+///
+/// # Arguments
+/// * `id` - The ID of the asset.
+/// * `spender` - The account that is allowed to spend the tokens.
+/// * `value` - The number of tokens to decrease the allowance by.
+///
+/// # Returns
+/// Returns `Ok(())` if successful, or an error if the operation fails.
+#[inline]
+pub fn decrease_allowance(_id: AssetId, _spender: AccountId, _value: Balance) -> Result<()> {
+	// let allowance = assets::allowance(id, owner, spender)?;
+	// assets::cancel_approval(id, spender.clone())?;
+	// assets::approve_transfer(id, spender, value)
+	Ok(())
+}
+
+/// 2. PSP-22 Metadata Interface:
+/// - token_name
+/// - token_symbol
+/// - token_decimals
+
+/// Returns the token name for a given asset ID.
+///
+/// # Arguments
+/// * `id` - The ID of the asset.
+///
+/// # Returns
+/// The name of the token as a byte vector, or an error if the operation fails.
+#[inline]
+pub fn token_name(id: AssetId) -> Result<Vec<u8>> {
+	assets::token_name(id)
+}
+
+/// Returns the token symbol for a given asset ID.
+///
+/// # Arguments
+/// * `id` - The ID of the asset.
+///
+/// # Returns
+///  The symbol of the token as a byte vector, or an error if the operation fails.
+#[inline]
+pub fn token_symbol(id: AssetId) -> Result<Vec<u8>> {
+	assets::token_symbol(id)
+}
+
+/// Returns the token decimals for a given asset ID.
+///
+/// # Arguments
+/// * `id` - The ID of the asset.
+///
+/// # Returns
+///  The number of decimals of the token as a byte vector, or an error if the operation fails.
+#[inline]
+pub fn token_decimals(id: AssetId) -> Result<u8> {
+	assets::token_decimals(id)
+}
+
+// /// 3. Asset Management:
+// /// - create
+// /// - start_destroy
+// /// - destroy_accounts
+// /// - destroy_approvals
+// /// - finish_destroy
+// /// - set_metadata
+// /// - clear_metadata
+//
+// /// Create a new token with a given asset ID.
+// ///
+// /// # Arguments
+// /// * `id` - The ID of the asset.
+// /// * `admin` - The account that will administer the asset.
+// /// * `min_balance` - The minimum balance required for accounts holding this asset.
+// ///
+// /// # Returns
+// /// Returns `Ok(())` if successful, or an error if the creation fails.
+// // pub fn create(id: AssetId, admin: impl Into<MultiAddress<AccountId, ()>>, min_balance: Balance) -> Result<()> {
+// // 	assets::create(id, admin, min_balance)
```

Comment:
Functionality regarding asset management should be taken care of in #92. How this will evolve into regarding your other point about whether creating an asset should be called in the constructor is something that will have to be looked at when working that issue. Included it in the issue for reminder.